### DISCUSSION TITLE
How will Arroyo integrate nicely with code versioning/branching(git) and CI/CD?

### DISCUSSION THREAD

**Discussion by let-me-cook** [ArroyoSystems/arroyo#335] [Q&A]
Title

**Comment by dis-sid** [comment]
Hi,
This one is a basic need we have for such tools, I couldn't imagine using dbt without git for instance,
How multiple persons can work on the same project/pipelines ? is it possible to have pipeline queries collectively reviewed, versioned and automatically applied (deployed) ?
